tests/test_eclblackoilpvt_eval.cpp

```diff
@@ -0,0 +1,908 @@
+// -*- mode: C++; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*-
+// vi: set et ts=4 sw=4 sts=4:
+/*
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 2 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+
+  Consult the COPYING file in the top-level source directory of this
+  module for the precise wording of the license and the list of
+  copyright holders.
+*/
+/*!
+ * \file
+ *
+ * \brief This is the unit test for the black oil PVT classes
+ *
+ * This test requires the presence of opm-parser.
+ */
+#include "config.h"
+
+#if !HAVE_ECL_INPUT
+#error "The test for the black oil PVT classes requires eclipse input support in opm-common"
+#endif
+
+#include <iostream>
+#include <iomanip>
+#include <cmath>
+#include <limits>
```

Comment:
the order of includes in opm-material is in decreasing order of abstractness, i.e., headers from the standard library should be included last. (this rule makes missing includes a little bit more visible.)